packages/twenty-server/src/workspace/messaging/services/gmail-full-sync.service.ts

```diff
@@ -173,7 +189,7 @@ export class GmailFullSyncService {
     );
 
     if (messagesToSave.length > 0) {
-      this.saveMessagesAndCreateContactsService.saveMessagesAndCreateContacts(
+      await this.saveMessagesAndCreateContactsService.saveMessagesAndCreateContacts(
```

Comment:
Missing await here... Which caused exceptions to not be caught and crashes the worker